catalog/spatial/geocoding/spatial-geocoding-admin-module/src/main/java/org/codice/ddf/spatial/admin/module/service/Geocoding.java

```diff
@@ -66,11 +66,7 @@ private void registerMbean() {
           LOGGER.debug("Re-registered Geocoding Configuration MBean");
         }
       } catch (Exception e) {
-        // objectName is not always non-null because new ObjectName(...) can throw an exception
-        LOGGER.info(
-            "Could not register MBean [{}].",
-            objectName != null ? objectName.toString() : Geocoding.class.getName(),
-            e);
+        LOGGER.info("Could not register MBean [{}].", objectName, e);
```

Comment:
❗️ This form will not print out the exception stack trace as we are also passing `objectName`. Use string concatenation for `objectName` instead.
❓ Should we trace the exception at debug instead of info?

Reply:
The log level is `info` in the master branch. I would prefer to leave it that way unless less this logging statement is known to spam the log.

Reply:
I am not suggesting changing the whole log level, keep that log but log the exception stack trace at debug level not to pollute the info level. BTW, as it is right now, the exception and the stack trace is not logged. The version of the `info()` method being called expects parameters to format the string. It is not the same as info(String, Throwable) which will log the message and the stack trace of the exception,

Reply:
@paouelle Actually it looks like slf4j is smart enough to pick off the throwable as the last argument. 

![image](https://user-images.githubusercontent.com/9013416/34536163-8fd1e74c-f081-11e7-8918-63ef3d56e11c.png)

Reply:
Looks like they finally updated their code to do that. 👍  
Also different behavior than log4j.
Sorry about all those comments then 😉

Reply:
That being said, I am with @Lambeaux and I am not sure we want to pollute other levels than debug with exception stack traces.